### PR TITLE
chore: add format to scheduler events

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #5299

### CODE REVIEW

File: packages/backend/src/analytics/LightdashAnalytics.ts [hunk 1]
```diff
@@ -498,6 +498,7 @@ export type SchedulerUpsertEvent = BaseTrack & {
         cronExpression: string;
         cronString: string;
         resourceId: string;
+        format: 'csv' | 'image';
```
Comment (joaoviana) [inline packages/backend/src/analytics/LightdashAnalytics.ts:501]: is it worth it having an enum for this?

**Review by joaoviana — APPROVED** [review]
non-blocking comment!

File: packages/backend/src/analytics/LightdashAnalytics.ts (hunk 1 above)
Comment (ZeRego) [reply packages/backend/src/analytics/LightdashAnalytics.ts]: fixed :)

Comment (joaoviana) [reply packages/backend/src/analytics/LightdashAnalytics.ts]: :chefs kiss:

**Comment by joaoviana** [comment]
🚢 it!

**Review by joaoviana — APPROVED** [review]
🚢 !

**Pull request merged**